### ISSUE TITLE
Executing a meta-target just hangs

### ISSUE THREAD

**Issue by stephenchu** [tonyfischetti/sake#73]
Using version `1.0`, upon `sake combo` where `combo` is the name of a meta-target, it just seems to hang; if I removed the meta-target and just execute the individual tasks, it's fine:

```
> cat Sakefile
combo:
  help: combo
  uno:
    help: uno
    formula: echo "UNO"
  dos:
    help: dos
    formula: echo "DOS"
```
```
> sake -v combo
Call to check_integrity issued
Sakefile passes integrity test
Going to construct Graph
Adding 'dos'
Adding 'uno'
Nodes are built
Building connections
checking node dos for dependencies
checking node uno for dependencies
# hangs right here
```
I was using Python 2.7.14, pip installing `master-sake` to see this behavior. I can provide a Dockerfile to repro this if it isn't reproducible on your end.

**Comment by tonyfischetti** [comment]
Hmm, it also seems if you just run `sake` (without combo) it works, too

**Comment by tonyfischetti** [comment]
Fixed as of 6d4e726ad4932f
Thanks so much for bringing this to my attention :)

**Comment by tonyfischetti** [comment]
Let me know if you run into any problems with it. I'll be submitting to PyPI later today
